vendor/monkey-patch-ember-computed.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  let computedStore = new WeakMap();
```

Comment:
This now requires that `WeakMap` be implemented, which I don't think the library did previously. I'm pretty sure you could just change this to `new Ember.WeakMap()` because this should always be imported after `ember-weakmap`'s shims are imported because `index.js` calls the super `included` before doing its `app.import()`. Or if we don't care about browsers that don't implement `WeakMap`, then maybe it's time to remove `ember-weakmap`?

Reply:
Good catch on the weakmap, I didn't intent to regress like that. I guess since I already accidentally regressed [here](https://github.com/kellyselden/ember-macro-helpers/pull/260/files#diff-b773593af4b01d438ca5bdef65ebf731R1) and WeakMap works in IE11, I suppose we can just leave it?

Reply:
Yup, no objection here! I haven't touched a browser without WeakMaps in quite a few years :)